### PR TITLE
Add AMI installation instructions

### PR DESCRIPTION
**Pull request by LeeHampton**

### CODE REVIEW

File: getting-started/installation-ami.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Region | Image id
+--- | ---
+us-east-1 | ami-0ecf0cc105d5a8fc0
+us-east-2 | ami-0bd0ceaa391794533
+us-west-1 | ami-03645fd772b136a5a
+us-west-2 | ami-0566aa080793cac7c
+
+
+To launch the AMI, go to the `AMIs` section of your AWS EC2 Dashboard and select
+`Public Images` under the dropdown menu. Filter the image-id by the image id for your
```
Comment (dianasaur323) [inline getting-started/installation-ami.md]: do we want to use the same verbage for Image id (above) vs image-id vs image id?

File: getting-started/installation-ami.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+TimescaleDB is currently available as an Ubuntu 16.04 Amazon EBS-backed AMI. AMIs are
+distributed by region, and our AMI is currently available in `us-east-1`, `us-east-2`,
+`us-west-1`, and `us-west-2`. Note that this image is built to use an EBS attached volume
+rather than the default disk that comes with EC2 instances.
+
+See below for the image id corresponding to each region for the most recent TimescaleDB version:
+
+Region | Image id
+--- | ---
+us-east-1 | ami-0ecf0cc105d5a8fc0
```
Comment (dianasaur323) [inline getting-started/installation-ami.md]: on the top right in the AWS console when you select regions, it doesn't tell me if it's us-east-1 or us-east-2. It says N.Virginia. Should we include the associated region in plain English here?

**Comment by dianasaur323** [comment]
Ok, approved with 2 minor comments.

File: getting-started/installation-ami.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+settings, which are not optimal for most systems. See our [configuration] section for tips on
+optimally configuring postgresql.conf for your particular setup.
+
+>:TIP These AMIs are made for EBS attached volumes. This allows for snapshots, protection of
+data if the EC2 instance goes down, and dynamic IOPS configuration. You should choose an
+EC2 instance type that is optimized for EBS attached volumes. For information on choosing the right
+EBS optimized EC2 instance type, see the AWS [instance configuration page].
+
+[configuration]: /getting-started/configuring
+[instance-configuration-page]: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-ec2-config.html
```
Comment (solugebefola) [inline getting-started/installation-ami.md]: Did you test this locally?  Dashes vs. no dashes in the text.

File: getting-started/installation-ami.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+## Installing from an Amazon AMI (Ubuntu)
```
Comment (solugebefola) [inline getting-started/installation-ami.md]: Where is this page going to be located in the site hierarchy?

**Comment by dianasaur323** [comment]
@LeeHampton lgtm, thanks!

**Comment by LeeHampton** [comment]
@solugebefola I'm thinking this should go under the Linux installation page as another tab next to 'apt-ubuntu`, `apt-yum`, `Docker`, etc. I restructured the files and headers to mimic the other ubuntu- pages, but I'm not 100% sure what else needs to change to ensure it gets into that hierarchy. I also updated the link issue.

File: getting-started/installation-ubuntu-ami.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+Region | Image ID
+--- | ---
+us-east-1 (North Virginia) | ami-0ecf0cc105d5a8fc0
+us-east-2 (Ohio) | ami-0bd0ceaa391794533
+us-west-1 (North California) | ami-03645fd772b136a5a
+us-west-2 (Oregon) | ami-0566aa080793cac7c
+
+
+To launch the AMI, go to the `AMIs` section of your AWS EC2 Dashboard and select
```
Comment (stalltron) [inline getting-started/installation-ubuntu-ami.md]: Just for readability, can we make these bullets?

Comment (solugebefola) [reply getting-started/installation-ubuntu-ami.md]: They are in a table right now.  Is that not easy to read?

Comment (solugebefola) [reply getting-started/installation-ubuntu-ami.md]: They are in a table right now.  Is that not easy to read?

**Comment by LeeHampton** [comment]
Good call on the bullets @stalltron -- let me know how it looks now.

**Comment by LeeHampton** [comment]
@tgeisenheimer Added a paragraph around DB setup and pg_hba. Didn't go into too much detail because I don't think our documentation should be too prescriptive about security/ACL settings.

**Comment by LeeHampton** [comment]
@solugebefola is this good to merge or should I keep it as a branch for dev testing since it involves a UI tweak?

**Comment by solugebefola** [comment]
ok to merge
